### PR TITLE
[IMP] Add check Duplicate field in xml record

### PR DESCRIPTION
**Pull request by lescobarvx**
Check same field assigned 2 times in xml record.

### CODE REVIEW

**Comment by lescobarvx** [comment]
@moylop260 @pedrobaeza @max3903 
Please your feedback

**Comment by pedrobaeza** [comment]
You should check or at least put as known issues that duplicated fields across inherited views are not taken into account.

**Comment by lescobarvx** [comment]
@pedrobaeza 
the change is ready where if it finds an inherited view does not take into account

**Comment by pedrobaeza** [comment]
So it's working with inherited views or not?

**Comment by lescobarvx** [comment]
@pedrobaeza 
It doesn't work with inherited views because I haven't a viable solution to check them at this time

**Comment by pedrobaeza** [comment]
OK, then you should mention it in the description of the warning.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by lescobarvx** [comment]
@pedrobaeza 
Ready, I put an important note in the docstring of _check_duplicate_xml_fields method indicating that this check does not work with inherited views, that's OK?

**Comment by pedrobaeza** [comment]
Yeah, I see. You already have my :+1:.

Thanks for the work.

**Comment by lescobarvx** [comment]
@pedrobaeza 
you're welcome

**Comment by moylop260** [comment]
Please, add a test case with inherit

File: pylint_odoo/test_repo/broken_module/model_view_odoo2.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                </tree>
+            </field>
+            <field name="name">view.model.form4</field>
+        </record>
+
+        <!-- inherit view is not checked -->
+        <record id="view_model_form2" model="ir.ui.view">
+            <field name="name">view.model.form</field>
+            <field name="model">test.model</field>
+            <field name="inherit_id" ref="id_category_list"/>
```
Comment (lescobarvx) [inline pylint_odoo/test_repo/broken_module/model_view_odoo2.xml]: @moylop260 
Ready, added test case of inherited views.

Comment (moylop260) [inline pylint_odoo/test_repo/broken_module/model_view_odoo2.xml]: Duplicated and non duplicated with inherit here too pleaseplease

**Comment by lescobarvx** [comment]
@moylop260 @pedrobaeza 
Ready, 2 test case for inherited views: with duplicate fields and without duplicate fields

**Comment by moylop260** [comment]
Thanks
:+1:

**Pull request merged**